### DISCUSSION TITLE
Packet Sender UDP

### DISCUSSION THREAD

**Discussion by audionutt** [dannagle/PacketSender#346] [Q&A]
Why does Packet Sender show the UDP server disabled when in settings the UDP server is enabled with a specific port #  Thanks for the assist !

**Comment by dannagle** [comment]
Make sure the actual checkbox says it is enabled/disabled. That is what actually toggles it.

Filled in means enabled:
![image](https://github.com/dannagle/PacketSender/assets/1140573/80daafa9-783b-443e-9ade-9b199eb3928f)


You can also tap the server button on the bottom right to toggle it.
